Sources/RayTracer.Library/Mathematics/Vector3.cs

```diff
@@ -14,6 +14,8 @@ namespace RayTracer.Library.Mathematics;
     public readonly float Y;
     
     public readonly float Z;
+    
+    public float Length => (float)Math.Sqrt(X * X + Y * Y + Z * Z);
```

Comment:
Let's have this as a method to match the System.Numerics.Vector3 API

Reply:
Do you want it like this? 
public float Length => new System.Numerics.Vector3(X,Y,Z).Length();
?